app/assets/javascripts/katello/sync_management/sync_management.js

```diff
@@ -346,15 +346,19 @@ KT.content_actions = (function () {
   };
 })();
 
-var ids = [];
-$.each(KT.repo_status, function (repo_id, status) {
-  if (status.is_running) {
-    ids.push(repo_id);
-    KT.content.draw_syncing(repo_id, status.progress.progress, status.sync_id);
-  }
-});
-KT.content.reset_products(KT.repo_status);
-KT.content_actions.addSyncing(ids);
+KT.content.populate_repo_status = function () {
```

Comment:
I think that you have to define the function further up in KT.content (declared in line 1) and then you have to put it into the return block in line 209 of this file https://github.com/Katello/katello/pull/11156/files#diff-30d150bd0c3aaa9d5e785d962c40718a880598b9d58677f958882c76e9bc5e49R209. 

And then, if you want to access this function, you can do it down here by addressing it via `KT.content.populate_repo_status()`

I have not tested on Katello nightly, but if the code is as ancient as you say, Katello 4.13.1 might still be good enoght :D

Reply:
What I get inside my debugging tool: 
![image](https://github.com/user-attachments/assets/01928e19-f1a4-4697-a389-729b29f7eb12)

Reply:
I have tested the above suggestion. That works for me (on Katello 4.13.1).